### PR TITLE
[FEAT] new SpinningIcon component

### PR DESCRIPTION
**Pull request by nirpeled**
Export a new component to allow spinning a FA icon.

NOTE: I wonder if it will be better to just pass a boolean to the original component and then have the logic internally instead of exporting two separated component - I leave it to you guys :)

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (jgcmarins) [inline .idea/vcs.xml]: I guess this file can be added to .gitignore

Comment (nirpeled) [reply .idea/vcs.xml]: Yeah - my bad, will remove it now

Comment (jgcmarins) [reply .idea/vcs.xml]: @nirpeled no problem 😄

**Comment by rturk** [comment]
@jgcmarins @sibelius what do you think?

**Pull request merged**